actors/evm/src/interpreter/instructions/storage.rs

```diff
@@ -12,11 +12,7 @@ pub fn sload(
     let location = state.stack.pop();
 
     // get from storage and place on stack
-    let value = match system.get_storage(location)? {
-        Some(val) => val,
-        None => U256::zero(),
-    };
-    state.stack.push(value);
+    state.stack.push(system.get_storage(location)?);
```

Comment:
error? Doesnt evm zero for everything?
Ot is it internal now?

Reply:
This is a fatal "something is really broken" error (same as before).

Reply:
Ok, do we have a test for this?

Reply:
No? This is a "the HAMT is corrupted" error. I.e., there's a bug.

Reply:
ok, no need to bother then.